xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -176,9 +182,46 @@ rendering.macro.taglist.notags=As you and your team add tags to the content, thi
 #### Confluence Bridges in the generic pro macro package
 
 ## Content Report Table macro
+rendering.macro.content-report-table.name=Content Report Table
+rendering.macro.content-report-table.description=Provides a content report in table format, based on labels.
+rendering.macro.content-report-table.parameter.labels.name=Label
+rendering.macro.content-report-table.parameter.labels.description=Label(s) to report on
+rendering.macro.content-report-table.parameter.spaces.description=Space(s) to report on
+rendering.macro.content-report-table.parameter.maxResults.name=Max results
+rendering.macro.content-report-table.parameter.maxResults.description=Maximum Number of Pages
 contentReportTable.header.modified=Modified
 
 ## Contributors macro
+rendering.macro.contributors.name=Contributors
+rendering.macro.contributors.description=COntributors Confluence Bridge macro to show the contributors of a given document.
```

Comment:
```suggestion
rendering.macro.contributors.description=Contributors Confluence Bridge macro to show the contributors of a given document.
```

Reply:
Also, since this macros is part of the generic module, we should give it a description and name independent of the concept of confluence